src/objectModel.ts

```diff
@@ -7706,7 +7702,7 @@ export interface IPhaseFilterTable {
 
 export class PhaseFilter implements IPhaseFilter {
     index: number
-    new?: number
+    _new?: number
```

Comment:
This case should probably be handled during the code generation phase. Otherwise we'd have to fix it manually every time.

Reply:
Yes, that's where I actually made the fix.